### PR TITLE
Allow maximum 5 groups in a session workspace

### PR DESCRIPTION
**Pull request by adamabeshouse**
Implements UI to select which ones

image

clicking on "354 more groups..." brings up a modal

image

image

image

image

image

### CODE REVIEW

**Comment by schultzn** [comment]
Wow. This looks really nice. 

I don’t like the term “in workspace” - can you maybe replace that with something less technical, such as “in group comparisons”?

File: src/pages/groupComparison/GroupComparisonStore.ts [hunk 1]
```diff
@@ -88,6 +90,12 @@ export default class GroupComparisonStore {
         return this.appStore.isLoggedIn;
     }
 
+    @autobind
+    @action
+    public setWorkspaceGroups(map:ObservableMap<boolean>) {
+        this.isGroupInWorkspace = map.toJS();
```
Comment (alisman) [inline src/pages/groupComparison/GroupComparisonStore.ts]: this sounds like a boolean but is actually a map.  why not a method that takes group as param and looks in map? also, why toJS?

Comment (adamabeshouse) [reply src/pages/groupComparison/GroupComparisonStore.ts]: sure, I can split it into a map and a clearer getter method

toJS just makes a copy of it, I think it's a little cleaner that way than using the map thats passed in which the caller still has access to. and it's not necessary to be an observable map on this end, so toJS is good

**Comment by adamabeshouse** [comment]
image

**Comment by adamabeshouse** [comment]
lets force open the modal if there are more than 5 groups in the workspace

**Comment by adamabeshouse** [comment]
image

image

**Comment by adamabeshouse** [comment]
image

**Pull request closed without merging**